scripts/install

```diff
@@ -4,10 +4,19 @@ source _common.sh
 source /usr/share/yunohost/helpers
 source _ynh_mysql_connect_as.sh
 
-phpflags="--define apc.enable_cli=1"
+#=================================================
+# INITIALIZE AND STORE SETTINGS
+#=================================================
 
+phpflags="--define apc.enable_cli=1"
 ynh_app_setting_set --app=$app --key=phpflags --value="$phpflags"
 
+enable_notify_push="0"
```

Comment:
This is supposed to come from the manifest, right ??

Reply:
This is no longer the case, as we want to simplify the installation. The advanced settings should still be in the config panel.